.github/workflows/create-jira-issue.yml

```
@@ -1,154 +1,156 @@
+                      echo "template=chore-task.yml" >> $GITHUB_OUTPUT
+                    elif echo "${{ toJson(github.event.issue.labels) }}" | grep -q '✅ test'; then
+                      echo "type=test" >> $GITHUB_OUTPUT
+                      echo "template=test-task.yml" >> $GITHUB_OUTPUT
+                    else
+                      echo "type=feature" >> $GITHUB_OUTPUT
+                      echo "template=feature-task.yml" >> $GITHUB_OUTPUT
+                    fi
+
+            -   name: Clean Issue Title (for Jira Summary)
+                id: clean
+                run: |
```

actionlint (1.7.8)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)